com.gitlab.sane_project.XSane.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.gitlab.sane_project.XSane
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```

Comment:
Update runtime to 19.08

Reply:
@scx 19.08 is a requirement for new Flathub submissions.

Reply:
@AsciiWolf 
> 19.08 is a requirement for new Flathub submissions.

Last month it wasn't a problem:
> 18.08 will be supported for one more year

> I don't think it's a blocker per se for the time being. I feel it will bite us when 18.08 is EOL though.

Anyway, I'll update the `Freedesktop` runtime to `19.08` tomorrow.
The main problem is that I don't have scanners from various vendors so I need to borrow them from my friends. Once a year is not a problem, but doing it every month or two is problematic. And we already know that proper testing is necessary, because flatpak support for scanners is not so great.
https://github.com/flathub/flathub/pull/1111#issuecomment-520837857

Reply:
There is a bug in Freedesktop 19.09 that prevents such an update.
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/921

Reply:
The bug was "fixed" in early December, but in fact it persisted until January 13.
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/tags/freedesktop-sdk-19.08.6

Just for comparison:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/freedesktop-sdk-19.08.5/elements/components/pciutils.bst
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/freedesktop-sdk-19.08.6/elements/components/pciutils.bst#L23-38

Unfortunately, **KDE** `5.13` is still affected.